bosh-release/README.md

```diff
@@ -4,7 +4,18 @@ A BOSH release for deploying WSO2 API Manager 2.1.0 on BOSH Director:
 
 ## Quick Start Guide
 
-1. Install BOSH CLI v2.x from [bosh.io](http://bosh.io/)
+1. Install dependency tools.
+
+  a) Install BOSH CLI v2.x from [bosh.io](http://bosh.io/)
+  
+  b) Install Ruby.
+
+      $ sudo apt install ruby 
+    
+  c) Install VirtualBox.
+
+      $ sudo apt install virtualbox
```

Comment:
I think its better to consider OSX and other Linux platforms when providing instructions.

Reply:
+1